spring-boot-autoconfigure/src/main/java/org/springframework/boot/autoconfigure/jms/activemq/ActiveMQConnectionFactoryFactory.java

```diff
@@ -59,6 +61,9 @@
 	private <T extends ActiveMQConnectionFactory> T doCreateConnectionFactory(
 			Class<T> factoryClass) throws Exception {
 		T factory = createConnectionFactoryInstance(factoryClass);
+		MutablePropertyValues propertyValues =
+				new MutablePropertyValues(this.properties.getProperties());
+		new RelaxedDataBinder(factory).bind(propertyValues);
```

Comment:
Unfortunately, not.

There is no `properties` property on `ActiveMQConnection`.  `PooledConnectionFactory` _does_ have a `properties` property, but the setter has the side-effect of constructing a brand new instance of `ActiveMQConnection` and so that is not a viable path.